web/src/app/core/services/authentication.service.ts

```diff
@@ -15,8 +15,8 @@ import { filter, concatMap, switchMap, first, takeUntil, tap } from 'rxjs/operat
 import { DeviceDetectorService } from 'ngx-device-detector';
 
 // Dashboard hub models
-import { ProfileModel, LoginAuditModel } from '../../shared/models/index.model';
 import { ActivityService } from './activity.service';
+import { ProfileModel, LoginAuditModel } from '../../shared/models/index.model';
```

Comment:
Grouping import should also follow the alphabetical ordering. It should be fixed with lint rule

Reply:
I think you are looking at old code, GitHub has marked your comment as `outdated`.

<img width="856" alt="Screenshot 2019-05-18 14 44 55" src="https://user-images.githubusercontent.com/624760/57970619-891f9f00-797b-11e9-8c2a-3cfa8179d6dd.png">


Please see https://github.com/DashboardHub/PipelineDashboard/pull/1237/files#diff-9fb615cefc2705778e04e2ef31bec93bR18

Reply:
okay looks good 👍